valet-key/ValetKey.Web/Controllers/SasController.cs

```diff
@@ -1,32 +1,33 @@
-﻿
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-namespace ValetKey.Api.Controllers
-{
-    using System;
-    using System.Configuration;
-    using System.Diagnostics;
-    using System.Net;
-    using System.Net.Http;
-    using System.Web.Http;
-    using Azure.Storage;
-    using Azure.Storage.Blobs;
-    using Azure.Storage.Sas;
-    using Microsoft.Azure;
+﻿using Azure.Storage;
+using Azure.Storage.Blobs;
+using Azure.Storage.Sas;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using System;
+using System.Diagnostics;
+using System.Net;
+using System.Net.Http;
 
-    public class ValuesController : ApiController
+namespace ValetKey.Web.Controllers
+{
+    [ApiController]
+    public class SasController : ControllerBase
     {
         private readonly BlobServiceClient blobServiceClient;
         private readonly string blobContainer;
+        private IConfiguration configuration;
 
-        public ValuesController()
+
+        public SasController(IConfiguration configuration)
         {
-            this.blobServiceClient = new BlobServiceClient(CloudConfigurationManager.GetSetting("Storage"));
-            this.blobContainer = ConfigurationManager.AppSettings["ContainerName"];
+            this.configuration = configuration;
+            this.blobServiceClient = new BlobServiceClient(configuration.GetSection("AppSettings:StorageConnectionString").Value);
+            this.blobContainer = configuration.GetSection("AppSettings:ContainerName").Value;
```

Comment:
I know it sounds weird, but honestly, I'd just hardcode this.  Making it a variable, but not really telling anyone to change it, and at the same time telling them to create it with exactly this specific name doesn't seem consistent.  Simplify the config by removing it and just hardcoding it where it's needed below.  Hopefully you'll get to a point where the only config value is just the blob storage account url (and not even a connection string)

Reply:
Yeah.. I started to clean this up and realized we just need the blob endpoint.